### PR TITLE
new application class

### PR DESCRIPTION
**Pull request by CcoleCode**
testing edits

### CODE REVIEW

**Review by CcoleCode — COMMENTED** [review]
checking the added comments

**Comment by BaileyPatt** [comment]
Sweet, I can see it on my end

**Comment by AnatalioNickles** [comment]
yo, looks good for now. 1545

File: JavaLeague/src/Application.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+		infoMethod();
+		deleteMethod();
+		searchMethod();
+		exitMethod();
+		
+		//
+		do {
+		//run method menu until the exit option is chosen
+
+
```
Comment (AnatalioNickles) [inline JavaLeague/src/Application.java:22]: // is this working? 1546

**Review by AnatalioNickles — CHANGES_REQUESTED** [review]
only made one edit to this current code

File: JavaLeague/src/Application.java (hunk 1 above)
Comment (CcoleCode) [reply JavaLeague/src/Application.java:22]: i can see your comment

**Pull request merged**